packages/deployment/upgrade-test/Dockerfile

```diff
@@ -11,94 +14,115 @@ WORKDIR /usr/src/agoric-sdk/
 COPY ./start_ag0.sh ./upgrade-test-scripts/
 COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./upgrade-test-scripts/
 SHELL ["/bin/bash", "-c"]
+# this is the only layer that starts ag0
 RUN . ./upgrade-test-scripts/start_ag0.sh
 
+## UPGRADE LAYER
 ## this is agoric-upgrade-8 aka pismoA
 FROM ghcr.io/agoric/agoric-sdk:29 as agoric-upgrade-8
 ARG BOOTSTRAP_MODE
+# ??? why doesn't this set UPGRADE_TO ?
```

Comment:
The upgrade from pismoA to pismoB was a hot fix (without a governance proposal), so it doesn't use any `tx gov software-update`.